laws/src/main/scala/cats/mtl/laws/discipline/KindTransformerTests.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats
+package mtl
+package laws
+package discipline
+
+import cats.kernel.laws.discipline.catsLawsIsEqToProp
+import org.scalacheck.Prop.{forAll => ∀}
+import org.scalacheck.Arbitrary
+import org.typelevel.discipline.Laws
+
+trait KindTransformerTests[F[_], G[_]] extends Laws {
+  implicit val ktInstance: KindTransformer[F, G]
+
+  def laws: KindTransformerLaws[F, G] = KindTransformerLaws[F, G]
+
+  def kindTransformer[A](
+      implicit arbFA: Arbitrary[F[A]],
+      arbGA: Arbitrary[G[A]],
+      arbFF: Arbitrary[F ~> F],
+      eqGA: Eq[G[A]]
+  ): RuleSet =
```

Comment:
I think  `List` is a good candidate for `F` here (as it yields many simple `F ~> F` implementations such as `drop(n)`, `take(n)`)

Reply:
Do we have to write that arbitrary ourself, or does Scalacheck provide one out of the box, like it does for Function (with the help of Cogen).

Reply:
we have/had to write it ourselves